client/app/templates/components/game-client.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-12">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span class="glyphicon glyphicon-comment"></span> Client
+        </div>
+        <div class="panel-body">
+          <ul class="chat">
+            {{#each messages as |msg|}}
+              <li class="{{if msg.isServer "left" "right"}} clearfix">
```

Comment:
Yeah it was copied from a chat app, haha so I liked the message container and quickly templated it up removing a lot of it. The 'server'/'you' thing was more to ensure there was a difference so I preferred the swapping so I know it was the server since it just echos.

What I would like to see going forward is to restyle the chat body tag into something more like a console that has a prompt, and the footer could blend in with the console to appear like a prompt at the bottom of the window.

so you end up with something that is like

```
> your command
timestamp - server response
> enter your command
```

Don't now thinking out loud not an expert on mud UI. But you could style with colors etc.